editor/js/Strings.js

```diff
@@ -778,7 +782,11 @@ function Strings( config ) {
 			'viewport/info/triangles': 'Triangles',
 			'viewport/info/sample': 'Échantillon',
 			'viewport/info/samples': 'Échantillons',
-			'viewport/info/rendertime': 'Temps de rendu'
+			'viewport/info/rendertime': 'Temps de rendu',
+
+			'script/title/vertexShader': 'Shader de sommet',
+			'script/title/fragmentShader': 'Shader de fragments',
```

Comment:
At least in French, it is unusual to translate the Vertex and Fragment Shader, see:

https://fr.wikipedia.org/wiki/Shader

Reply:
good